frequency-chain/testnet-faucet
# Form can be valid regardless of order filling out

## Goal
This adjusts the faucet form validation to allow the submit button to be enabled regardless of the order of clicking captcha or filling in an address.  Address validation isn't performed until `focusout` fires. A little trickier to fix than at first blush.

Also did a yarn update to fix a browserslist warning.  Thanks to @JoeCap08055 for the nudge to turn this into an issue.

Closes #27 

## Checklist


## How To Test the Changes

1. Clone the pr branch
2. Set PUBLIC_DEMO_MODE=true in `client/.env` and launch:
```shell
cd client
yarn install --freeze-lockfile && yarn dev
```
3. Then verify that you can click Captcha and then fill out the address, and the Submit button is enabled, as well as vice versa.
4. Verify that the address is still validated correctly.
